syntax = "proto2";

import "groups/wyckoff_position.proto";
import "groups/irrep.proto";

package magnon.groups;

message AtomicOrbital {
    optional WyckoffPosition wyckoff_position = 1;

    optional Irrep site_symmetry_irrep = 2;
}
